src/actions/activity.js

```diff
@@ -3,31 +3,31 @@
 import { ACTIVITY_MESSAGES_LIMIT } from '../global/Constants';
 import { type ActivityLogMessage } from '../types/ActivityLogMessage';
 
-type MessageAddedAction = { +type: 'MESSAGE_ADDED', +message: ActivityLogMessage };
+type MessageAddedAction = { +type: 'MESSAGE_ADDED', +messages: Array<ActivityLogMessage> };
```

Comment:
It's a bit unclear, why action is called `MessageAdded`, but takes messages array. Please make naming and semantics to be in line.